doc/classes/Particles2D.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Particles2D" inherits="Node2D" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		2D Particle emitter
+		2D particle emitter.
 	</brief_description>
 	<description>
-		Particles2D is a particle system 2D [Node] that is used to simulate several types of particle effects, such as explosions, rain, snow, fireflies, or other magical-like shinny sparkles. Particles are drawn using impostors, and given their dynamic behavior, the user must provide a visibility bounding box (although helpers to create one automatically exist).
+		2D particle node used to create a variety of particle systems and effects. Particles features an emitter that generates some number of particles at a given rate.
```

Comment:
`Particles` -> `[code]Particles[/code]`

Reply:
Should it be `[code]Particles2D[/code]`?

Reply:
Good suggestions.  I debated the voice because the particles aren't the subject node, they're emitted *by* the node.